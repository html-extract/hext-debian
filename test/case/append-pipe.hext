<a class:trim:prepend("'"):append("' is the class"):'class' />
