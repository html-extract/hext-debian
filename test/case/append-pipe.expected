{"class":"'no spaces' is the class"}
{"class":"'spaces' is the class"}
